homeassistant/components/garmin_connect/sensor.py

```diff
@@ -181,9 +181,13 @@ async def async_update(self):
             self._state = data[self._type] // 60
         elif "Mass" in self._type or self._type == "weight":
             self._state = round((data[self._type] / 1000), 2)
-        elif (
-            self._type == "bodyFat" or self._type == "bodyWater" or self._type == "bmi"
-        ):
+        elif self._type in [
+            "bodyFat",
+            "bodyWater",
+            "bmi",
+            "floorsAscended",
```

Comment:
This addition seems like a new feature. Is that correct?

Reply:
It round the values of some more sensors, it doesn't brake anything.
I can add some of these things in separate PR's but to save your precious times I combined some.
I also wanted to implement DataCoordinator, but that was a big impact in code, so left that one for later.
Thanks.

Reply:
We want to separate new features from fixes. Each thing should be a separate PR. Please break it out.

Reply:
Removed